client/src/components/TaskStatusChart/TaskStatusChart.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import styles from "./TaskStatusChart.module.css";
+import { PieChart, Pie, Cell, Legend } from "recharts";
+import { PropTypes } from "prop-types";
+
+const RADIAN = Math.PI / 180;
+const renderCustomizedLabel = ({
+  cx,
+  cy,
+  midAngle,
+  innerRadius,
+  outerRadius,
+  percent,
+  index,
+}) => {
+  const radius = innerRadius + (outerRadius - innerRadius) * 0.5;
+  const x = cx + radius * Math.cos(-midAngle * RADIAN);
+  const y = cy + radius * Math.sin(-midAngle * RADIAN);
+
+  return (
+    <text
+      x={x}
+      y={y}
+      fill="white"
+      textAnchor={x > cx ? "start" : "end"}
+      dominantBaseline="central"
+    >
+      {`${(percent * 100).toFixed(0)}%`}
+    </text>
+  );
+};
+
+const COLORS = ["#629632", "#F6BE00", "#FF0000"];
+
+export const TaskStatusChart = ({ tempData }) => {
+  return (
+    <div>
```

Comment:
The wrapping div here can be removed (unless some type of wrapping div container is needed here later when merging the rest of the analytics page together)

Reply:
Resolved: <div> is added for the header for each Piechart.